plugins/ftdidmx/FtdiWidget.h

```diff
@@ -126,6 +142,36 @@ class FtdiWidget {
       return m_name + " with serial number : " + m_serial +" ";
     }
 
+    /** @brief Get Widget available interface count **/
+    int GetInterfaceCount();
+
+    /* @brief
+     * Build a list of available ftdi widgets.
+     * @param widgets a pointer to a vector of FtdiWidgetInfo objects.
+     */
+    static void Widgets(std::vector<FtdiWidgetInfo> *widgets);
+
+ private:
+    std::string m_serial;
```

Comment:
Can you space everything so private is indented by one space, then the private code indented by two spaces.